tests/unit/core/reducers/test_search.js

```diff
@@ -66,13 +67,19 @@ describe('search reducer', () => {
 
     it('sets the results', () => {
       const { results } = getNextState();
-      expect(results).toEqual([{ slug: 'foo' }, { slug: 'food' }]);
+      expect(results).toEqual([
+        createInternalAddon({ slug: 'foo' }),
```

Comment:
I won't mark each one but a couple more of these need to change to `{ ...fakeAddon, slug: '...' }`